### ISSUE TITLE
Meaning of adjustedtime

### ISSUE THREAD

**Issue by ArashJavan** [Kitware/VeloView#98]
Can plz anybody tell me what is the real meaning o the field "adjustedtime".
Which unit does this field have?
Which time reference?

Thanks

**Comment by rageobi** [comment]
Hey @ArashJavan . Where you able to figure out the answer? Please do let know if you did.

**Comment by ArashJavan** [comment]
@rageobi not really, so I decided to write my own [Veloparser](https://github.com/ArashJavan/veloparser).

**Comment by rageobi** [comment]
Ah, right. I have a collected dataset in the veloview format.  And was looking to understand the headers. Cheers, thanks anyway
